benchmarks/polys.py

```diff
@@ -71,7 +70,6 @@ def make_poly(self, n):
         g = d * (2 + x ** (n + 1) + sum([y[i] ** (n + 1) for i in range(n)]))
         return f, g, d, syms
 
-
 class _QuadraticNonMonicGCD(_GCDExample):
```

Comment:
Why are these lines deleted? Usually there should be two lines between top level statements like `def` and `class`.

Reply:
I thought there should be one line between subclasses, I will correct myself.